.github/workflows/pull-request.yml

```diff
@@ -250,7 +250,7 @@ jobs:
 
       - name: Filter files
         id: get-lint-files
-        run: node ./script/github-actions/filter-files-changed.js ${{ steps.get-files-changed.outputs.files_modified }}
+        run: node ./script/github-actions/filter-files-changed.js ${{ steps.get-files-changed.outputs.files }}
```

Comment:
Gets all new, updated, and removed files instead of just modified files: https://github.com/trilom/file-changes-action#outputs